koku/api/utils.py

```diff
@@ -124,6 +127,12 @@ def last_month_start(self):
         last_month = self.this_month_start - self.one_day
         return last_month.replace(day=1)
 
+    @property
+    def materialized_view_month_start(self):
+        """Datetime of midnight on the first of the month where materialized summary starts."""
+        summary_month = self.this_month_start - relativedelta(months=Config.MASU_RETAIN_NUM_MONTHS - 1)
+        return summary_month
+
```

Comment:
I'm not sure this is the right place for this. DateHelper is supposed to only care about helping us get our date math correct. It doesn't really need to know about the app in any way. I think this is a case of yes we _can_ do it this way, but I'm not sure we _should_ do it this way. :slightly_smiling_face:  

I also think there's a simpler solution.

In the validator, we make this change:
```
-         if value >= dh.last_month_start.date() and value <= dh.today.date():
+        MAX_QUERY_DAYS=60
+        if value >= dh.n_days_ago(dh.today, MAX_QUERY_DAYS).date() and value <= dh.today.date():
```

I think I'd prefer to not use MASU_RETAIN_NUM_MONTHS. I think we're coalescing around the day being the basic unit we calculate around. For that reason, I think the MASU_RETAIN_NUM_MONTHS feature probably needs a refactor and maybe a small redesign to update it to our current direction. It might be good to open a new issue to maybe do a spike to investigate if that's something we'd want to do.